examples/memory.rs

```diff
@@ -0,0 +1,89 @@
+use wasmer::{
+    imports, wat2wasm, Extern, Function, Instance, Memory, MemoryType, Module, NativeFunc, Pages,
+    Store, Table, TableType, Type, Value,
+};
+use wasmer_compiler_cranelift::Cranelift;
+use wasmer_engine_jit::JIT;
+
+// this example is a work in progress:
+// TODO: clean it up and comment it
```

Comment:
Still actual ;-).

Reply:
Yeah I was intending to ship it like that 😆 that's a `TODO` for another time.  I can do it before shipping but I didn't want to block on it